### PR TITLE
DOC modify itemized form in  readme.rst and readme.ja.rst

### PR DESCRIPTION
**Pull request by shopetan**
issue  https://github.com/clear-code/pikzie/issues/15

Modify itemized form.
If this PR need modifications , please point out me .

### CODE REVIEW

File: README.ja.rst [hunk 1]
```diff
@@ -94,16 +94,16 @@ git::
 
   import pikzie
   import テスト対象のモジュール
-  
+
```
Comment (kou) [inline README.ja.rst:97]: Why these spaces are removed? Is the change needed for itemization?

File: README.ja.rst [hunk 2]
```diff
@@ -468,14 +468,11 @@ pikzie.priority(priority)
...
-  * aztmさん
-
-    * バグレポートをしてくれました。
-
-    * ebuildを作ってくれました。
-      http://diary.atzm.org/20081201.html#p01
-
-  * Hideo Hattoriさん
+- aztmさん
+ 	バグレポートをしてくれました。
```
Comment (kou) [inline README.ja.rst]: Why do you change to definition list from nested itemization?
It seems that this change breaks the original meaning.

The original: https://github.com/clear-code/pikzie/blob/master/README.ja.rst#%E8%AC%9D%E8%BE%9E
The changed: https://github.com/shopetan/pikzie/blob/fixreadme/README.ja.rst#%E8%AC%9D%E8%BE%9E

The original means that "aztm" (oh, it's a typo... He is "atzm" not "aztm"...) has two contributions. But the changed means that "aztm" has only one contribution.

**Comment by kou** [comment]
Thanks for your try!
I've commented some points.

**Comment by shopetan** [comment]
Thank you for your comments.

> Why these spaces are removed? Is the change needed for itemization?

No it is not. 
My Editor has removed the tab in the auto-completion. sorry.

> Why do you change to definition list from nested itemization?
> It seems that this change breaks the original meaning.

Sorry. Fix this miss and typo.

**Comment by kou** [comment]
> My Editor has removed the tab in the auto-completion. sorry.

I understand. Could you remove these changes from this pull request? Because these changes aren't related to itemization.

> Fix this miss and typo.

Ah, sorry. I confused you. The typo fix should be done in another pull request. Because the typo fix isn't related to itemization. Could you send another pull request that includes only the typo fix?

**Comment by shopetan** [comment]
> Could you remove these changes from this pull request? Because these changes aren't related to itemization.

I understand.  remove these changes.

> The typo fix should be done in another pull request. 

Okay. 
Once I close this PR, and again build a PR.

Thank you.

**Pull request closed without merging**